tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -472,11 +471,16 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
        |WindowEvent -  java.awt.event
        |""".stripMargin,
     compat = Map(
-      ">=2.13.0" -> """|Widget -  example
-                       |Window -  java.awt
-                       |WindowPeer -  java.awt.peer
-                       |WithFilter -  scala.collection
-                       |""".stripMargin
+      "2.13" -> """|Widget -  example
+                   |Window -  java.awt
+                   |WindowPeer -  java.awt.peer
+                   |WithFilter -  scala.collection
+                   |""".stripMargin,
+      "3" -> """|Widget -  example
+                |Window -  java.awt
+                |WindowPeer -  java.awt.peer
+                |WithFilter($0) - [A](p: A => Boolean, xs: Array[A]): WithFilter[A]
```

Comment:
That was weird for me, but turns out it's a legit method :thinking:

Reply:
Yeah, I, too, found this one suspicious, ha.